demo/index.html

```diff
@@ -1,81 +1,758 @@
+                                    </svg>
+                                </button>
+                                <span>Feed the cat</span>
+                            </li>
+                        </ul>
+                        <h3 id="kanban-board">
+                            <a href="#kanban-bord">Kanban board</a>
+                        </h3>
+                        <p>Drag and drop tasks into a Kanban board:</p>
+                        <ul id="kanban-list">
+                            <li>
+                                <p>You can switch tasks by dropping one on top of another.</p>
+                            </li>
+                            <li>
+                                <p>You can reorder tasks by dragging them up or down.</p>
+                            </li>
+                            <li>
+                                <p>You can move them to another column by dragging them into it.</p>
+                            </li>
+                        </ul>
+                        <div id="kanban-board">
+                            <div id="backlog" class="block">
+                                <h4>Backlog</h4>
+                                <ul>
+                                    <li class="task" draggable="true">
+                                        <span>Implement user authentication flow</span>
+                                    </li>
+                                    <li class="task" draggable="true">
+                                        <span>Write unit tests for API</span>
+                                    </li>
+                                </ul>
+                            </div>
+                            <div id="in-progress" class="block">
+                                <h4>In progress</h4>
+                                <ul>
+                                    <li class="task" draggable="true">
+                                        <span>Add error handling for login</span>
+                                    </li>
+                                </ul>
+                            </div>
+                            <div id="done" class="block">
+                                <h4>Done</h4>
+                                <ul>
+                                    <li class="task" draggable="true">
+                                        <span>Set up the database</span>
+                                    </li>
+                                    <li class="task" draggable="true">
+                                        <span>Create API endpoints for users</span>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </section>
+                </main>
+            </div>
+            <footer>
+                <span class="copyright">
+                    &copy; <span class="current-year"></span>, drag-drop-touch-js
```

Comment:
There is nothing in this file that requires copyrighting, let's remove this footer.